src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -258,71 +258,78 @@ def resolve_dependencies(self, main_ctx: AnalyzeContext, sbom_path: str) -> dict
 
         deps_resolved: dict[str, DependencyInfo] = {}
 
-        build_tool = main_ctx.dynamic_data["build_spec"]["tool"]
-        if not build_tool or isinstance(build_tool, NoneBuildTool):
-            logger.info("Unable to find a valid build tool.")
+        build_tools = main_ctx.dynamic_data["build_spec"]["tools"]
+        if len(build_tools) == 0:
```

Comment:
You can just use `if build_tools` in Python to check if a list is empty or not, no need to check the length.